app/controllers/provider/sessions_controller.rb

```diff
@@ -28,12 +28,12 @@ def create
       create_user_session!(strategy&.authentication_provider_id)
       flash[:notice] = 'Signed in successfully'
 
-      AuditLogService.call("Signed in: #{current_user.id}/#{current_user.username} #{current_user.first_name} #{current_user.last_name}")
```

Comment:
Is this log removed because now there are audit logs for the "UserSession" model?

Reply:
Yes, exactly. Now we audit `UserSession` and also `User#password_digest`, so we don't need augit logs for login/logout and user password change